### ISSUE TITLE
Please give example of Timestamps.txt

### ISSUE THREAD

**Issue by yashb042** [HugoM25/Character_Versus_Video_Generator#2]
General advice is to add a sample project always where the user can just simply go and run it. I am more than happy to contribute once I run a sample project.

**Comment by HugoM25** [comment]
Yeah I added an example file [here](https://drive.google.com/file/d/1A6_AwdgUMDysjyuZJkRSKs5-SPNPhYoY/view?usp=share_link). If you want to test the program you can run the commands : 
`python main.py --scriptPath ExampleRes/Scripts/example1.txt --resFolder ExampleRes/`
Or also : 
`python main.py --scriptPath ExampleRes/Scripts/example2.txt --resFolder ExampleRes/`
